src/__tests__/pathToPointer.ts

```diff
@@ -8,4 +8,6 @@ test('pathToPointer', () => {
   expect(pathToPointer(['paths', 'foo~users'])).toEqual('#/paths/foo~0users');
   expect(pathToPointer([])).toEqual('#');
   expect(pathToPointer([''])).toEqual('#/');
+  expect(pathToPointer(['paths', '/user/{userId}'])).toEqual('#/paths/~1user~1{userId}');
+  expect(pathToPointer(['paths', '/user/{userId}/~foo'])).toEqual('#/paths/~1user~1{userId}~1~0foo');
```

Comment:
Can we pull this test over from marc's branch?

`expect(pathToPointer(['paths', '/users/{id}', 'get'])).toEqual('#/paths/~1users~1{id}/get');`

Reply:
He did :).